serialmfg/serial_resources/process_entry.py

```diff
@@ -136,10 +168,65 @@ def add_boolean(self, dataset_name, value, expected_value):
         Returns:
         - API response for adding boolean data
         """
-        dataset = serial.Datasets.get(dataset_name, "CHECKBOX", self.process_id)
+        dataset = None
+        try:
+            dataset = serial.Datasets.get(dataset_name, "CHECKBOX", self.process_id)
```

Comment:
Should this be `BOOLEAN`?

Reply:
No, since it's pulling the dataset's data_type, which is still on the old system